java/src/jmri/jmrix/acela/AcelaSensorManager.java

```diff
@@ -33,16 +35,22 @@ public AcelaSensorManager(AcelaSystemConnectionMemo memo) {
      * {@inheritDoc}
      */
     @Override
+    @Nonnull
     public AcelaSystemConnectionMemo getMemo() {
         return (AcelaSystemConnectionMemo) memo;
     }
 
     /**
-     * Create a new sensor if all checks are passed. System name is normalized to
-     * ensure uniqueness.
+     * {@inheritDoc}
+     * <p>
+     * System name is normalized to ensure uniqueness.
+     *
+     * @return null if the system name is not in a valid format (TODO change that to throw an exception, Spotbugs)
      */
     @Override
-    public Sensor createNewSensor(String systemName, String userName) {
+    @Nonnull
+    @SuppressFBWarnings(value = "NP_NONNULL_RETURN_VIOLATION", justification = "Null result signals input error, change to exception TODO")
```

Comment:
Suppressing a warning that we have explicitly indicated needs to be fixed hides the need to fix it. Please don't do this.